src/Appwrite/Auth/Auth.php

```diff
@@ -73,7 +74,7 @@ class Auth
     public const TOKEN_EXPIRATION_LOGIN_SHORT = 3600;         /* 1 hour */
     public const TOKEN_EXPIRATION_RECOVERY = 3600;            /* 1 hour */
     public const TOKEN_EXPIRATION_CONFIRM = 3600 * 1;         /* 1 hour */
-    public const TOKEN_EXPIRATION_PHONE = 60 * 15;            /* 15 minutes */
```

Comment:
Why did we remove phone? 

Also if we were using `TOKEN_EXPIRATION_PHONE ` in other places, we need to replace it now

Reply:
Checked code, not used anywhere. Removed because phone and magic both does same 6digit 15min expiry OTP, which comes from `TOKEN_EXPIRATION_OTP`